DSharpPlus/Clients/DiscordClient.Dispatch.cs

```diff
@@ -1970,7 +1971,8 @@ internal async Task OnWebhooksUpdateAsync(DiscordChannel channel, DiscordGuild g
         internal async Task OnStickersUpdatedAsync(IEnumerable<DiscordMessageSticker> newStickers, JObject raw)
         {
             var guild = this.InternalGetCachedGuild((ulong)raw["guild_id"]);
-            var oldStickers = new ReadOnlyDictionary<ulong, DiscordMessageSticker>(guild._stickers);
+            var oldStickers = guild._stickers.ToImmutableDictionary();
```

Comment:
why not `var oldStickers = new ConcurrentDictionary<ulong, DiscordSticker>(guild._stickers);`

Reply:
I'm yet to get back around to this PR to fix that

Reply:
oky <3

Reply:
yea please change that.